flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/CodeGenUtils.scala

```diff
@@ -124,14 +124,27 @@ object CodeGenUtils {
 
   private val nameCounter = new AtomicLong
 
-  def newName(name: String): String = {
-    s"$name$$${nameCounter.getAndIncrement}"
+  def newName(context: CodeGeneratorContext = null, name: String): String = {
```

Comment:
This one has a default value for `context`, but `newNames` does not have, is there any rational behind this?

Reply:
Parameter section with *-parameter is not allowed to have default arguments

Reply:
My point is that do you really need to have a default value, IIUC, we want all places to pass `context`?

Reply:
Get it, removed